src/components/SystemMenu/SystemMenu.jsx

```diff
@@ -122,20 +137,30 @@ function SystemMenu({ showTutorial }) {
             }
             <HorizontalDelimiter />
 
-            <button type="button" onClick={() => onClick(console)}>
+            <button type="button" onClick={() => onClick(openGuiInBrowser)}>
+              Open GUI in Browser
+            </button>
+
+            <HorizontalDelimiter />
+
+            <button type="button" onClick={() => onClick(showLuaConsole)}>
               Toggle console
               {' '}
               <span className={styles.shortcut}>~</span>
             </button>
+
             <button type="button" onClick={() => { onClick(nativeGui); }}>
               Toggle native GUI
               {' '}
               <span className={styles.shortcut}>F1</span>
             </button>
-            {/*              <button onClick={saveChange}>
+
+            {/* <button onClick={saveChange}>
```

Comment:
Not sure if we want to keep this comment?

Reply:
I really don't know either.  It shows the intention of bringing back the button for saving changes to the profile, but at this stage I don't even know if we want to. 

I would be ok with removing it